CMakeLists.txt

```diff
@@ -46,16 +46,12 @@ add_custom_target(assemble_single_header ALL DEPENDS ${CMAKE_CURRENT_SOURCE_DIR}
 ## TESTS/EXAMPLES/HELPERS
 ################################################################################
 
-if(${DOCTEST_WITH_MAIN_IN_STATIC_LIB} AND NOT CMAKE_CURRENT_SOURCE_DIR STREQUAL CMAKE_SOURCE_DIR)
+if(${DOCTEST_WITH_MAIN_IN_STATIC_LIB})
```

Comment:
concerning this change, please leave it as, it makes development inside IDE easier because they detect the target and thus they detect the .cpp file too.

Reply:
fair enough :)